packages/manager/modules/vps/src/dashboard/tile/information/information.html

```diff
@@ -0,0 +1,101 @@
+<oui-tile
```

Comment:
Changes concerns : 
- replacement of `<i>`by `<span>`
- some `::`
- remove of extra `data-`

Reply:
Simple question: I wonder why we remove the extra `data-`?

Reply:
Hmm they concerns ovh-ui-kit components attributes, so we don't need it. What do you think ?

Reply:
It's optional indeed. 
I add `data-` to non standard html attribute in order to recognize them more easily. But this is my point of view :smile: